### DISCUSSION TITLE
Is there no way to select multiple lines and then expand/collapse those sections with the keyboard?

### DISCUSSION THREAD

**Discussion by smellythief** [vincode-io/Zavala#248] [Support]
Am I crazy that the left and right arrow keys don't collapse and expand, respectively, the selected sections? I think this should be the expected behavior. Do we really have to click on each lines arrow separately? This is really slowing me down and may be enough to have me keep looking for a usable outline app.

**Comment by vincode-io** [comment]
Bike and OmniOutliner are both fine apps and are "useable". Do me a favor, purchase one of them, and move along.
